Simple go test is broken

I try to run tests as usual but it returns error now:
```
$ go test ./...
?   	github.com/src-d/sourced-ce/cmd/sourced	[no test files]
?   	github.com/src-d/sourced-ce/cmd/sourced/cmd	[no test files]
?   	github.com/src-d/sourced-ce/cmd/sourced/compose	[no test files]
?   	github.com/src-d/sourced-ce/cmd/sourced/compose/file	[no test files]
?   	github.com/src-d/sourced-ce/cmd/sourced/compose/workdir	[no test files]
?   	github.com/src-d/sourced-ce/cmd/sourced/dir	[no test files]
?   	github.com/src-d/sourced-ce/cmd/sourced/format	[no test files]
?   	github.com/src-d/sourced-ce/cmd/sourced/release	[no test files]
--- FAIL: TestInitLocalTestSuite (0.00s)
    --- FAIL: TestInitLocalTestSuite/TestChangeWorkdir (0.00s)
        require.go:794:
            	Error Trace:	common.go:49
            	            				suite.go:112
            	Error:      	Received unexpected error:
            	            	fork/exec ../build/sourced-ce_darwin_amd64/sourced: no such file or directory
            	Test:       	TestInitLocalTestSuite/TestChangeWorkdir
    --- FAIL: TestInitLocalTestSuite/TestWithInvalidWorkdir (0.00s)
        require.go:794:
            	Error Trace:	common.go:49
            	            				suite.go:112
            	Error:      	Received unexpected error:
            	            	fork/exec ../build/sourced-ce_darwin_amd64/sourced: no such file or directory
            	Test:       	TestInitLocalTestSuite/TestWithInvalidWorkdir
FAIL
FAIL	github.com/src-d/sourced-ce/test	0.025s
```

--- a/test/commander.go
+++ b/test/commander.go
@@ -1,3 +1,5 @@
+// +build integration
+
 package test
 
 import (

--- a/test/common.go
+++ b/test/common.go
@@ -1,3 +1,5 @@
+// +build integration
+
 package test
 
 import (

--- a/test/init_local_test.go
+++ b/test/init_local_test.go
@@ -1,3 +1,5 @@
+// +build integration
+
 package test
 
 import (

--- a/test/init_orgs_test.go
+++ b/test/init_orgs_test.go
@@ -1,3 +1,5 @@
+// +build integration
+
 package test
 
 import (

--- a/test/superset.go
+++ b/test/superset.go
@@ -1,3 +1,5 @@
+// +build integration
+
 package test
 
 import (

